views/pages/EditChartPage.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from 'react'
+import { useParams, useNavigate } from 'react-router-dom'
+
+import ChartForm from '../forms/ChartForm'
+import { editChart, getChart } from '../fe-utils/fetchUtil'
+import { routes } from '../routes/routes'
+
+const EditChartPage = ({ classes, user }) => {
+  const { chart_id } = useParams()
+  const navigate = useNavigate()
+  const [chart, setChart] = useState()
+  const handleSubmit = async (e, formValues) => {
+    e.preventDefault()
+    const { data } = await editChart(chartID, formValues)
+
+    if (data.modifiedCount === 1) navigate(routes.chart(chart_id))
```

Comment:
this should return the updated chart instead of the modifiedCount. I think we can also assume if `await editChart` does not throw an error that the request was successful and we can navigate to the chart page.